setup.cfg

```diff
@@ -1,5 +1,5 @@
 [tool:pytest]
```

Comment:
There is a python3.6.core above, as well as iocage.8.gz, can we remove those binaries?